Resync `css-animations` from WPT Upstream

Ahmad-S792 opened the pull request:
#### 5fdb29dc297e1e49a9592e499ec906e29d953f91
<pre>
Resync `css-animations` from WPT Upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=301044">https://bugs.webkit.org/show_bug.cgi?id=301044</a>
<a href="https://rdar.apple.com/162943266">rdar://162943266</a>

Reviewed by Matthieu Dubet.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/19250d046daa69748ece904fb05c768713e045aa">https://github.com/web-platform-tests/wpt/commit/19250d046daa69748ece904fb05c768713e045aa</a>

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animate-font-size-with-margin-override.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animate-with-background-color-oklch-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animate-with-background-color-oklch-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animate-with-background-color-oklch-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animate-with-background-color-oklch-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animate-with-background-color-oklch-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/crashtests/chrome-bug-404743651.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/crashtests/chrome-bug-405795970.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/crashtests/chrome-bug-415627003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/neutral-var-keyframe-cycle-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/neutral-var-keyframe-cycle-registered-additive-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/neutral-var-keyframe-cycle-registered-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parent-after-change-style-pseudo-element-flicker.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-range-end-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-range-end-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-range-start-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-range-start-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-range-start-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-range-start-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-end-computed-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-end-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-end-invalid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-end-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-end-valid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-end-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-shorthand-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-shorthand.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-start-computed-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-start-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-start-invalid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-start-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-start-valid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-start-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-end-computed-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-end-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-end-invalid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-end-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-end-valid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-end-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-shorthand-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-shorthand.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-start-computed-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-start-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-start-invalid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-start-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-start-valid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-start-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-shorthand.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-shorthand.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-timeline-parsing.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-timeline-parsing.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-type-computed.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-type-computed.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-type-invalid.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-type-invalid.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-type-valid.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-type-valid.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/keyframe-selectors-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/keyframe-selectors.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/301776@main">https://commits.webkit.org/301776@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/bf959e0db559e226e40d42ebe6ed243b11582393

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/127020 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/46656 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/37681 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/134024 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/78585 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/2f8452c8-f613-49e8-98fe-562d1e8211c2) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/47273 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/55183 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/96653 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/64676 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/5f99a507-4995-4b70-a00f-147d4a7dd6a7) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/129968 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/37835 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/113651 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/77163 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/a4f1148d-2096-4c3f-9742-dbac89ea92b4) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/36699 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/31796 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/77416 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/107686 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/32134 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/136550 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/53675 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/41336 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/105168 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/54179 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/110009 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/104859 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/26740 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/50380 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/28751 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/51194 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/53607 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/52846 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/56180 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/54605 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

mdubet reviewed (approved):
All the deletion is coming from upstream https://github.com/web-platform-tests/wpt/commit/6be489684532622464b50b547ccf4a5eb392cb42

LGTM

Ahmad-S792 committed:
Resync `css-animations` from WPT Upstream
https://bugs.webkit.org/show_bug.cgi?id=301044
rdar://162943266

Reviewed by Matthieu Dubet.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/19250d046daa69748ece904fb05c768713e045aa

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animate-font-size-with-margin-override.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animate-with-background-color-oklch-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animate-with-background-color-oklch-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animate-with-background-color-oklch-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animate-with-background-color-oklch-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animate-with-background-color-oklch-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/crashtests/chrome-bug-404743651.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/crashtests/chrome-bug-405795970.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/crashtests/chrome-bug-415627003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/neutral-var-keyframe-cycle-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/neutral-var-keyframe-cycle-registered-additive-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/neutral-var-keyframe-cycle-registered-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parent-after-change-style-pseudo-element-flicker.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-range-end-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-range-end-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-range-start-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-range-start-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-range-start-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-range-start-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-end-computed-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-end-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-end-invalid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-end-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-end-valid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-end-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-shorthand-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-shorthand.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-start-computed-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-start-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-start-invalid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-start-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-start-valid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-exit-range-start-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-end-computed-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-end-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-end-invalid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-end-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-end-valid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-end-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-shorthand-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-shorthand.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-start-computed-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-start-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-start-invalid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-start-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-start-valid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-range-start-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-shorthand.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-shorthand.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-timeline-parsing.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-timeline-parsing.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-type-computed.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-type-computed.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-type-invalid.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-type-invalid.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-type-valid.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/animation-trigger-type-valid.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/keyframe-selectors-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/keyframe-selectors.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-animations/w3c-import.log:

Canonical link: https://commits.webkit.org/301776@main
